client/components/jetpack/backup-delta/index.jsx

```diff
@@ -42,12 +57,9 @@ class BackupDelta extends Component {
 					</div>
 				) }
 			</div>
-		);
-	}
-
-	render() {
-		return <div className="backup-delta">{ this.renderRealtime() }</div>;
-	}
-}
+			);
```

Comment:
I'm still seeing this in the rendered output. Maybe you've got a commit that needs pushed?

Reply:
You're right. I forgot to push the change 🤦 . Thanks.